### DISCUSSION TITLE
How can I add orderList with letter or roman character

### DISCUSSION THREAD

**Discussion by truonglocbinh** [ueberdosis/tiptap#1841] [General Questions] [answered]
Hi, I've use tiptap and I can make orderList with number like below
```
1. xx
2. xx
```
But I want to change number to letter or roman character like below
```
a. xx
b. yy
```
```
I. xx
II. xx
```
How can I do this. Please help me.

**Comment by cadars** [comment, marked as answer]
To change how the list markers look like, specify the `list-style-type` you want in CSS:
https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type

But to change how you input ordered lists, you would need to extend the `ordered-list` extension.
